src/integrate/orchestrator.rs

```diff
@@ -731,16 +962,16 @@ events_keys = ["*"]
         );
 
         let mut labels = HashMap::new();
-        labels.insert("project".to_string(), self.network_name.to_string()); 
+        labels.insert("project".to_string(), self.network_name.to_string());
 
         let config = Config {
             labels: Some(labels),
             image: Some(devnet_config.stacks_api_image_url.clone()),
             domainname: Some(self.network_name.to_string()),
-            tty: Some(true),
+            tty: None,
             exposed_ports: Some(exposed_ports),
             env: Some(vec![
-                format!("STACKS_CORE_RPC_HOST=stacks.{}", self.network_name),
+                format!("STACKS_CORE_RPC_HOST=stacks-node.{}", self.network_name),
```

Comment:
Non-blocking: Looks like the container names, like `stacks-node`, is being referenced in a few places. If you decided to change the container name, it may break this ENV var accidentally if you don't also update it. Might be worth putting the container names in global vars to keep it consistent and easy to change.

Reply:
Good idea, will follow up with a subsequent PR!